ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/BatchingParallelIngestionPipe.scala

```diff
@@ -38,7 +40,8 @@ object BatchingParallelIngestionPipe {
       .async
       .mapAsync(ingestingParallelism)(ingester)
       // Stage 6: Preparing data sequentially for throttled mutations in database (tracking the ledger-end, corresponding sequential event ids and latest-at-the-time configurations)
-      .conflate(tailer)
```

Comment:
it seems to me that in this form 2 things can backpressure:
* If downstream (InMemoryState updates and friends) is backpressuring
* If tail ingestion backpressures

I think the second was not intentional, right?

How about we send it to a conflating sink here like:
```
      .alsoTo(
        Flow[DB_BATCH]
          .conflate(tailer)
          .mapAsync(1)(ingestTail)
          .to(Sink.ignore)
      )
```
And then we do batching and/or buffering?

Reply:
> I think the second was not intentional, right?

But it was. The purpose of this PR is to still allow tail ingestion back-pressuring.

Reply:
Not sure about that alsoTo solution:
* The order of execution can be inverted if ingestTail is slow leading to in memory state updates ahead of the persisted ledger end
* If ingestTail is very slow, due to conflation, the above problem is amplified

Do we want to deliberately allow in memory state updates front-running persistence ledger end updates?

Reply:
Discussed offline that it is fine as proposed.